src/ptvsd/adapter/ide.py

```diff
@@ -223,6 +223,10 @@ def launch_request(self, request):
 
         self.session.spawn_debuggee(request, sudo, args, console, console_title)
 
+        if "RedirectOutput" in self.session.debug_options:
```

Comment:
This should also check the plain request for `"redirectOutput": true,`, in the launch/attach json.

Reply:
We don't actually support "redirectOutput" as an option anywhere else right now (launcher, pydevd) - it's strictly a debug property.